fbw-a380x/src/wasm/systems/a380_systems/src/air_conditioning/local_controllers/trim_air_drive_device.rs

```diff
@@ -128,41 +121,18 @@ impl<const ZONES: usize, const ENGINES: usize> TrimAirDriveDevice<ZONES, ENGINES
         acs_overhead: &impl AirConditioningOverheadShared,
         should_close_taprv: bool,
         hot_air_id: usize,
+        pneumatic: &impl PackFlowValveState,
     ) -> bool {
         acs_overhead.hot_air_pushbutton_is_on(hot_air_id)
             && !self.active_channel.has_fault()
-            && !matches!(self.fault, Some(TaddFault::BothChannelsFault))
+            && ((pneumatic.pack_flow_valve_is_open(1)) || (pneumatic.pack_flow_valve_is_open(2)))
```

Comment:
Unnecessary parenthesizes:
```suggestion
            && (pneumatic.pack_flow_valve_is_open(1) || pneumatic.pack_flow_valve_is_open(2))
```